UserService/app/src/main/java/com/bookmyshow/userservice/AppConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.bookmyshow.userservice;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfig {
+
```

Comment:
Let us create a separate package called "config" & add configuration files. 
We will rename this file as "GenericConfig"